app/components/admin/participants/table_row.html.erb

```diff
@@ -0,0 +1,27 @@
+<tr class="govuk-table__row nhsuk-table__row">
+  <td class="govuk-table__cell nhsuk-table__cell">
+    <span class="nhsuk-table-responsive__heading">Full name</span>
```

Comment:
Why do we need these in every row, if we have them in the table head row?

Reply:
Those are normally hidden for the large screens, but the table completely switches when hitting tablet sizes.